cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/CassandraStorageSettings.java

```diff
@@ -198,6 +201,14 @@ public void setCradleInfoKeyspace(String cradleInfoKeyspace)
 	}
 
 
+	public String getSchemaVersion() {
+		return schemaVersion;
+	}
+
+	public void setSchemaVersion(String schemaVersion) {
+		this.schemaVersion = schemaVersion;

```

Comment:
I doubt if this is needed. Schema is changed via code and its version should be changed as well. Setting schema version from outside means "make Cradle API work with unexpected schema".

Reply:
Thanks for point it out. I thought I removed it, but I must have missed it when fixing a bad commit. It's removed now.